plugins/scroll-wheel-zoom-disable.js

```diff
@@ -1,17 +1,13 @@
 // @author         jonatkins
 // @name           Disable mouse wheel zoom
 // @category       Tweaks
-// @version        0.1.0
+// @version        0.1.1
 // @description    Disable the use of mouse wheel to zoom. The map zoom controls or keyboard are still available.
 
-
+/* exported setup --eslint */
```

Comment:
Let's move it to the end of file.

Reply:
IMHO lint exclusions should be stated at the start of the code as with all the other plugins.